README.md

```diff
@@ -81,13 +81,29 @@ releases of the toolkit.
     * GATK4 uses the [Conda](https://conda.io/docs/index.html) package manager to establish and manage the
       Python environment and dependencies required by GATK tools that have a Python dependency. This environment also 
       includes the R dependencies used for plotting in some of the tools. The ```gatk``` environment 
-      requires hardware with AVX support for tools that depend on TensorFlow (e.g. CNNScoreVariant). The GATK Docker image 
+      requires hardware with AVX support for tools that depend on TensorFlow. The GATK Docker image 
```

Comment:
Are there other tools in GATK that use Tensorflow?

Reply:
Yeah, you're right, the mention of Tensorflow will be obsolete once the CNN tools are removed. Pytorch, however, does apparently depend on the Intel MKL, which uses CPU-based features for acceleration:

https://www.reddit.com/r/MachineLearning/comments/iap6yo/discussion_pytorch_favors_intel_against_amds/